vector/src/main/java/im/vector/activity/RiotAppCompatActivity.kt

```diff
@@ -293,4 +296,11 @@ abstract class RiotAppCompatActivity : AppCompatActivity() {
         ConsentNotGivenHelper(this, savedInstanceState)
                 .apply { addToRestorables(this) }
     }
+
+    @SuppressLint("ObsoleteSdkInt")
+    protected fun applyScreenshotSecurity() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.ICE_CREAM_SANDWICH) {
```

Comment:
min API is set to 16 so no need to test against ICE_CREAM_SANDWICH (14). @SuppressLint can also be removed